ucm2/codecs/qcom-lpass/wsa-macro/SpeakerEnableSeq.conf

```diff
@@ -5,6 +5,6 @@ EnableSequence [
 	cset "name='WSA_RX1 INP0' RX1"
 	cset "name='WSA_COMP1 Switch' 1"
 	cset "name='WSA_COMP2 Switch' 1"
-	cset "name='WSA_RX0 Digital Volume' 68"
-	cset "name='WSA_RX1 Digital Volume' 68"
+	cset "name='WSA_RX0 Digital Volume' 84"
```

Comment:
Won't be better to remove this settings from the EnableSequence and keep the initailization only in the BootSequence ?

Reply:
Makes sense, will update the PR with suggested changes.

Reply:
Sorry for the delay, branch is now pushed with suggested change.